cmd/utils.go

```diff
@@ -1082,9 +1093,51 @@ func getStackLabels(config *RootCommandConfig) (map[string]string, error) {
 		}
 	}
 
+	if config.Buildah {
+		buildahDigest, err := getBuildahDigest(labels["dev.appsody.stack.id"], imageName, config)
+		if err != nil || buildahDigest == "" {
+			config.Warning.log(err)
+			return labels, nil
+		}
+		labels[appsodyStackKeyPrefix+"digest"] = buildahDigest
+		config.Debug.log("Digest label successfully added")
+	}
+
 	return labels, nil
 }
 
+func getBuildahDigest(idKey string, image string, config *RootCommandConfig) (digest string, err error) {
+	splitImage := strings.Split(image, ":") //Split the image to extract the image name and version separately
+	if len(splitImage) != 2 {
+		return "", errors.New("Error retrieving image name and tag used for build. The image digest will not be added as a label")
+	}
+	imageName := splitImage[0]    //The image name minus the version
+	imageVersion := splitImage[1] //The version of the image used
+	cmdName := "buildah"
```

Comment:
Can you add a comment here specifying the limation of the buildah version and why we have to use this command instead?